frameworks/cassandra/universe/marathon.json.mustache

```diff
@@ -154,7 +154,6 @@
     "TASKCFG_ALL_REMOTE_SEEDS": "{{service.remote_seeds}}",
     "EXECUTOR_URI": "{{resource.assets.uris.executor-zip}}",
     "BOOTSTRAP_URI": "{{resource.assets.uris.bootstrap-zip}}",
-    "SCHEDULER_URI": "{{resource.assets.uris.scheduler-zip}}",
     "CASSANDRA_URI": "{{resource.assets.uris.cassandra-tar-gz}}",
     "LIBMESOS_URI": "{{resource.assets.uris.libmesos-bundle-tar-gz}}",
```

Comment:
While in here, can also remove this LIBMESOS_URI envvar as it shouldn't be passed to tasks either (only used by scheduler)